packages/svelte-urx/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import type {
```

Comment:
```suggestion
import {
```

TSDX brings an older prettier version (and I don't want to go down the rabbit hole of upgrading it), which chokes on `import type` - run `yarn lint` to see that. I believe using plain import will work as well.